### ISSUE TITLE
🧹 Post-migration: s3 buckets left from the removed apps

### ISSUE THREAD

**Issue by ymao2** [ministryofjustice/analytical-platform#1794]
### User Story

This is the follow-up ticket for https://github.com/ministryofjustice/data-platform/issues/1757

This ticket is ticket holder for 
### All the s3 buckets which linked to the apps removed from above ticket 

- alpha-app-adjudicatons-webapp
- alpha-app-anvil-access-tool
- alpha-auto-dash
- alpha-app-ccmbpt-directory-app
- alpha-app-cfo-activity-hubs-lot-allocation
- alpha-app-commuter-timeline
- alpha-app-commuter-sandbox
- alpha-app-companies-house-api-tool
- alpha-app-ctsc-call-data-explorer
- alpha-app-dash-website
- alpha-app-data-discovery
- alpha-app-data-discovery-dev
- alpha-eu-exit-contract
- alpha-app-evidence-repository-data
- alpha-app-scorecard-form-dev
- alpha-app-hmcts-moj-crown-consolidation-webapp
- alpha-app-hmps-retention-app
- alpha-app-inflation-tool
- alpha-app-innovation-partner-lot-allocation
- alpha-app-judicial-wm
- alpha-app-kpi-s3-proxy
- alpha-app-laa-cm-db
- alpha-app-laa-fas-qc-dsa
- alpha-app-laa-cm-db
- alpha-app-laa-special-reasons-nlp
- alpha-app-legalaiddatatool
- alpha-app-legalaidtools
- alpha-msps-monitoring-dashboard
- alpha-app-novel-drug-words
- alpha-app-segmentation-tool-probation
- alpha-app-pqip
- alpha-app-pp-reports-tool
- alpha-demographic-tables
- alpha-app-pp-reports-tool
- alpha-app-public-pp-reports
- alpha-app-reoffending-evidence-repository-app
- alpha-app-sentencing-remarks-data
- alpha-court-travel-time-analysis
- alpha-upw-regional-backlog
- alpha-victims-bill-consultation
- alpha-app-wellbeing-comments-dashboard
- alpha-app-wellbeing-pulse
- alpha-yjb-stats
- alpha-app-workforce-planning
- alpha-rr-strategy-dashboard
- alpha-app-mags-returners-sankey

### Update the universal login page
Remove the isApp logic from the auth0’s universal login page once the gold-scorecard-app is removed from alpha

### Archive the following repos 
- workforce-planning-app
- rr-strategy
- mags-returners-sankey-s3

### Other tasks

- Should we put all those related documents somewhere? confluence and google spreadsheets
- How about trello board?

### Value / Purpose

- Make sure the s3 buckets are handled properly : keep the buckets if people still use the data there, remove redundant s3 buckets to save space. etc
- Make sure the documents were kept somewhere or removed rather than being forgotten 

### Definition of Done

- [x] The current status of s3buckets have been checked
- [ ] Universal page has been updated
- [ ] conclusion has been made for other parts in this ticket

**Comment by Gary-H9** [comment]
`interventions-deploy` can also be deleted as part of this activity.

**Comment by jamesstottmoj** [comment]
Last time the above S3 buckets were updated:

alpha-app-adjudicatons-webapp - 5/1/23
alpha-app-anvil-access-tool - 1/3/24
alpha-auto-dash - 15/9/21
alpha-app-ccmbpt-directory-app - 7/7/21
alpha-app-cfo-activity-hubs-lot-allocation - 12/8/20
alpha-app-commuter-timeline - 13/4/22
alpha-app-commuter-sandbox - 14/4/22
alpha-app-companies-house-api-tool - 28/9/22
alpha-app-ctsc-call-data-explorer - 16/8/21
alpha-app-dash-website - 18/1/22
alpha-app-data-discovery - 7/3/23
alpha-app-data-discovery-dev - 7/3/23
alpha-eu-exit-contract - 29/9/21
alpha-app-evidence-repository-data - 18/5/22
alpha-app-scorecard-form-dev - 26/2/24
alpha-app-hmcts-moj-crown-consolidation-webapp - 23/6/21
alpha-app-hmps-retention-app - empty
alpha-app-inflation-tool - 16/6/23
alpha-app-innovation-partner-lot-allocation - 17/2/21
alpha-app-judicial-wm - 7/9/21
alpha-app-kpi-s3-proxy - Doesn't exist
alpha-app-laa-cm-db - 3/5/22
alpha-app-laa-fas-qc-dsa - 8/4/22
alpha-app-laa-special-reasons-nlp - 29/6/23
alpha-app-legalaiddatatool - doesn't exist
alpha-app-legalaidtools - 14/9/22
alpha-msps-monitoring-dashboard - 25/10/23
alpha-app-novel-drug-words - 21/3/23
alpha-app-segmentation-tool-probation - 27/2/24
alpha-app-pqip - 14/3/22
alpha-app-pp-reports-tool - 16/10/21
alpha-demographic-tables - 20/6/22
alpha-app-public-pp-reports - 1/11/21
alpha-app-reoffending-evidence-repository-app - 13/10/21
alpha-app-sentencing-remarks-data - 06/12/19
alpha-court-travel-time-analysis - 1/2/24
alpha-upw-regional-backlog - 1/6/22
alpha-victims-bill-consultation - 30/8/23
alpha-app-wellbeing-comments-dashboard - 1/7/21
alpha-app-wellbeing-pulse - 28/6/21
alpha-yjb-stats - 7/5/21
alpha-app-workforce-planning - 5/7/21
alpha-rr-strategy-dashboard - 3/11/22
alpha-app-mags-returners-sankey - 3/11/21

**Comment by jamesstottmoj** [comment]
Running tool to archive the following buckets:
alpha-auto-dash
alpha-app-ccmbpt-directory-app
alpha-app-cfo-activity-hubs-lot-allocation
alpha-app-commuter-timeline
alpha-app-commuter-sandbox
alpha-app-companies-house-api-tool
alpha-app-ctsc-call-data-explorer
alpha-app-dash-website
alpha-eu-exit-contract
alpha-app-evidence-repository-data
alpha-app-hmcts-moj-crown-consolidation-webapp
alpha-app-hmps-retention-app
alpha-app-innovation-partner-lot-allocation
alpha-app-kpi-s3-proxy
alpha-app-laa-fas-qc-dsa
alpha-app-legalaiddatatool
alpha-app-pqip
alpha-app-pp-reports-tool
alpha-demographic-tables
alpha-app-public-pp-reports
alpha-app-reoffending-evidence-repository-app
alpha-app-sentencing-remarks-data
alpha-upw-regional-backlog
alpha-app-wellbeing-comments-dashboard
alpha-app-wellbeing-pulse
alpha-yjb-stats
alpha-app-workforce-planning
alpha-rr-strategy-dashboard
alpha-app-mags-returners-sankey

The following buckets have been updated in the past year so keeping for now:
alpha-app-adjudicatons-webapp
alpha-app-anvil-access-tool
alpha-app-data-discovery
alpha-app-data-discovery-dev
alpha-app-scorecard-form-dev
alpha-app-inflation-tool
alpha-app-laa-special-reasons-nlp
alpha-msps-monitoring-dashboard
alpha-app-segmentation-tool-probation

These buckets still have linked apps in control panel so have left alone:
alpha-app-judicial-wm
alpha-app-legalaidtools
alpha-app-novel-drug-words
alpha-victims-bill-consultation 

Universal login page has been updated to remove isForApp logic

**Comment by jamesstottmoj** [comment]
Reverted the isForApp logic in the Universal login page. This caused issues with a number of apps when attempting to access them.

**Comment by jamesstottmoj** [comment]
I have messages the data source admins on slack with regards to archiving the following buckets:

alpha-app-adjudicatons-webapp
alpha-app-data-discovery
alpha-app-inflation-tool
alpha-app-laa-special-reasons-nlp
alpha-msps-monitoring-dashboard

The following bucket has 1 admin but does not seem to be on slack and their email isn't listed:
alpha-app-data-discovery-dev

Potentially archive the bucket anyway?

**Comment by jamesstottmoj** [comment]
The following additional buckets have been archived:
alpha-app-adjudicatons-webapp
alpha-app-data-discovery
alpha-app-data-discovery-dev

The admins of the following buckets have requested they stay for now:
alpha-app-inflation-tool
alpha-msps-monitoring-dashboard

Waiting on an update on alpha-app-laa-special-reasons-nlp

**Comment by jamesstottmoj** [comment]
Archived alpha-app-laa-special-reasons-nlp
